### ISSUE TITLE
package not auto-updated

### ISSUE THREAD

**Issue by flack** [composer/packagist#200]
The following package

http://packagist.org/packages/midgard/createphp

is not auto-updated since 2012-07-26 13:57 UTC

If I look into the github admin settings, everything looks good, and when I click "Test hook" it tells me that the payload is deployed. Could it be that something doesn't work right on the packagist side of things? In my other repos, I have the same configuration and they work fine, it's just this one that gives problems

**Comment by ginsen** [comment]
I'm having the same issues, I don't know if it is a GitHub problem or not, my repo is ginsen/photosort and the auto-update service hook to Packagist don't work.

**Comment by TVart** [comment]
even if this this ticket is closed, i'd like to add this post witch can help such people.
I've had this problem to:
to fix it :
1 ) I've replaced url in https with git@ssh and the domain
2 ) I had domain filled like https://packagist.org/packages/owner/repo and set it to https://packagist.org

And... it works :)

**Comment by radiegtya** [comment]
it happened to me too.
What I was fixing was:
1. at packagist webhook services, rather than using Username with email (someone@gmail.com), use username only (someone)

and its work

**Comment by jaitaiwan** [comment]
Hey guys, thought I'd just pipe in and say watch out because similar issues can occur when you have a bitbucket repo because of this issue: [bitbucket.org/site/master/issue/7775](https://bitbucket.org/site/master/issue/7775/post-service-does-not-provide-useful%C2%A0%E2%80%A6). I've personally struggled with this for a long time before I realised where the issue lay.

If you're paying for bitbucket I highly suggest moving on from them as they have not been bothered to fix this issue in over 2 years.

Hope this helps!

**Comment by zot24** [comment]
FYI for me the issue was that I was forcing the version of the project on the version params in my composer.json file and I wasn't bumping it on each new tag

**Comment by compojoom** [comment]
I was fighting with this for an hour so here is what fixed it for me. As @Seldaek said - delete the package and create it again. After that it worked.

**Comment by nilopc** [comment]
@TVart method just works! Thanks!

**Comment by nick4fake** [comment]
This still does not work for most of my repos.

I am using username, token is correct, as well as url.

**Comment by phpstudyOne** [comment]
请问现在解决了这个问题没有？

**Comment by nglasl** [comment]
Throwing my +1 onto this. The auto-update will work 95% of the time, but fail the other 5% of the time. I tested this by adding a tag onto a repository, it would show under packagist. Then deleting the tag, it would remain under packagist. If you look under the integrations settings tab, there's a "last delivery failed, timeout" error. Pushing another tag, the tag appears under packagist, and the previous tag is now removed from packagist.
